policyengine_canada/parameters/gov/provinces/qc/tax/income/credits/medical_expenses/age_eligibility.yaml

```diff
@@ -0,0 +1,12 @@
+description: Quebec qualifies filers for the medical expenses tax credit at or above this age threshold.
+values:
+  2022-01-01: 18
+metadata:
+  unit: year
+  label: Quebec medical expenses tax credit age eligibility
+  reference:
+    - title: Revenu Quebec - Refundable Tax Credit for Medical Expenses (Line 462 - Point 1)
+      href: https://www.revenuquebec.ca/en/citizens/income-tax-return/completing-your-income-tax-return/how-to-complete-your-income-tax-return/line-by-line-help/451-to-480-refund-or-balance-due/line-462/point-1/
+    - title: Quebec I-3 - Taxation Act, Part I, Book IX, Title III, Chapter III.1, DIVISION II.18, 1029.8.117. (b)
+      href: https://www.legisquebec.gouv.qc.ca/en/document/cs/I-3?langCont=en#ga:l_i-gb:l_ix-gc:l_iii-gd:l_iii_1-ge:l_ii_18-h1
```

Comment:
I can't open this link - not sure if its just geoblocked in europe

Reply:
It might be blocked as I can open it